app/routes/api.sign.ts

```diff
@@ -1,25 +0,0 @@
-import sign from 'jwt-encode';
-import type { Route } from './+types/api.sign';
-
-// TODO: this endpoint should check for presence of OS jwt and valdate that sub in the OS jwt matches the sub in the request body
```

Comment:
If OS won't be able to fix the current issue, we will need to revert this PR and implement this TODO to make this safe.